site/src/pages/release-notes/1.27.2.mdx

```diff
@@ -0,0 +1,28 @@
+---
+date: 2024-02-14
+---
+
+## 🛠️ Bug fixes
+
+- <type://XdsEndpointGroup> now supports [Subset based Load Balancing](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/subsets),
+ which enables more sophisticated load balancing configurations. #5461
```

Comment:
```suggestion
 which enables more sophisticated load balancing configurations. #5461
- You can now specify a static `Cluster` configuration to define the `ClusterLoadAssignment` instead of receiving
 the `Cluster` resource from the control plane. #5468
```